README.rst

```diff
@@ -77,14 +77,14 @@ If you want to build your base docker image, then use:
     # All build instructions assume you're building from the root directory of the sagemaker-scikit-learn-container.
 
     # CPU
-    docker build -t sklearn-base:<Scikit-learn_version>-cpu-py<python_version> -f docker/<Scikit-learn_version>/base/Dockerfile.cpu --build-arg py_version=<python_version> .
+    docker build -t sklearn-base:<Scikit-learn_version>-<SageMaker_version>-cpu-py<python_version> -f docker/<Scikit-learn_version>-<SageMaker_version>/base/Dockerfile.cpu .
 
 ::
 
     # Example
 
     # CPU
-    docker build -t sklearn-base:0.20.0-cpu-py3 -f docker/0.20.0/base/Dockerfile.cpu --build-arg py_version=3 .
```

Comment:
Would it be better to use a variable for the version? This might help https://docutils.sourceforge.io/docs/ref/rst/directives.html

Reply:
Do you mean something like this?
```
.. |FRAMEWORK_VERSION| replace:: 0.23.1-1
```
I tried things like
```
.. parsed-literal::
docker build -t sklearn-base:|FRAMEWORK_VERSION|-cpu-py3 -f docker/|FRAMEWORK_VERSION|/base/Dockerfile.cpu .
```
and it works but then I can't get the result formatted as a code block. If I use a code block, I can't get the substitution to work. It's not straightforward to mix substitution and code block formatting.

Reply:
Talked to @balajitummala offline, and he helped me figure out how to make this work. Thanks @balajitummala.